packages/admin-ui/src/lib/core/src/shared/components/product-multi-selector-dialog/product-multi-selector-dialog.component.html

```diff
@@ -37,6 +37,12 @@
                 <span [title]="mode === 'product' ? item.productName : item.productVariantName">{{
                     mode === 'product' ? item.productName : item.productVariantName
                 }}</span>
+                <div *ngIf="mode === 'product'">
+                    <small>
+                        <span class="slug-label">slug:</span>
```

Comment:
This should be a translation token so that it works in other UI languages:

```html
<span class="slug-label">{{ 'common.slug' | translate }}:</span>
```

You could also consider using one of the built-in utility classes for the margin-right, e.g. `mr-1`, `mr-2` etc (whichever looks right to you). The benefit of this is that

1. no need to add another class to the stylesheet
2. we will be adding support to make those utility classes work for RTL languages like Arabic

Reply:
Thank you very much for the quick review, and thanks for the tips!
I applied the suggested changes in [#1b48276](https://github.com/vendure-ecommerce/vendure/pull/2461/commits/1b48276a7e2df53f5fee37e16be3bbb387f661e8).